ampere/README.md

```diff
@@ -0,0 +1,413 @@
+# Ampere Mt Jade (WiWynn) LinuxBoot Altra / Altra Max
+
+# LinuxBoot + Plan9 via u-root/cpu
+
+## Documentation
+
+LinuxBoot [documention](https://book.linuxboot.org/cpu/) of the Plan9 cpu/cpud commands implemented in [u-root/cpu](https://github.com/u-root/cpu)
+
+# Plan9 u-root/cpu Benefits
+
+This tool allows you to have a very small kernel in SPI-NOR and use ssh with 9P remote filesystems on a remote Arm64 system to have full access to any Arm64 tools without installing them locally. In the example below any tool that is installed on the Raspberry Pi will be availble to Mt Jade once the cpu to cpud connection is made.
+
+- Full access to remote filesystem
+- Firmware and OS bringup
+- Baseboard bring-up and debugging
+- Security testing e.g. [chipsec](https://github.com/chipsec/chipsec)
+- Factory diagnostics and manufacturing flow
+
+## LinuxBoot Docker Example
+
+There is an example project that will build LinuxBoot with 9P enabled and systemboot as the default for Mt Jade [here](https://gitlab.com/fryard/Ampere-Samples/-/tree/main/examples/edk2)
+
+## Plan9 Client cpu
+
+- Raspberry PI aarch64/Arm64 OS (Another Arm64 system *e.g. Mt Jade* would also work)
```

Comment:
We should update the comments/text in these sections to be Mt. Jade specific.

Reply:
Updated to include many Arm64 example clients and moved the README to the jade directory from the ampere directory.